web/cypress/integration/Trace/CreateAssertion.spec.ts

```diff
@@ -0,0 +1,89 @@
+import {createTest, deleteTest, testId} from '../utils/common';
+
+describe('Create Assertion', () => {
+  before(() => {
+    createTest();
+  });
+
+  after(() => {
+    deleteTest();
+  });
+
+  it('should create a basic assertion', () => {
+    cy.get(`[data-cy=test-url-${testId}]`).first().click();
+    cy.location('href').should('match', /\/test\/.*/i);
+
+    cy.get(`[data-cy^=test-run-result-]`).first().click();
+    cy.location('href').should('match', /resultId=.*/i);
+
+    cy.get('[data-cy=add-assertion-button]').click();
+    cy.get('[data-cy=create-assertion-form]', {timeout: 10000}).should('be.visible');
+
+    cy.get('[data-cy=item-selector-tag] + [role=img]').first().click();
+    cy.get('[data-cy=affected-spans-count]').should('have.text', 'Affects 7 spans');
```

Comment:
I think we should not set the expected text as `Affected 7 spans` because that might make the test brittle. What if we change our demo and now we have 8 spans? That would break the FE test.

I think it would be better to have something like:

```js
cy.get('[data-cy=affected-spans-count]').should('match', /Affects \d+ spans/);
```